README.md

```diff
@@ -9,9 +9,10 @@ You should see the main page of the system
 
 The following steps refer to the old sql file, will be change in 11/14
 ### How to login as Administrator (might need to set your phpmyadmin to no password)
-1. upload the sql file in database folder to phpmyadmin
+1. In db_connect.php, $user and $password should be set to your phpmyadmin user and password
 2. in phpmyadmin, add a database called final_project (db_connect.php line 5 dbname=final_project)
-3. in final_project, add a relation called administer (adminIn.php line 12)
-4. in administer, set Administer_ID and Password two attribute (adminIn.php line 12)
-5. in administer, set your own Administer_ID and Password
-6. go back to [adminpage](http://localhost/admin/adminSignIn.php) to test if the administer login works
+3. upload the sql file (located under database folder) into final_project database in phpmyadmin
+4. in final_project, add a relation called administer (adminIn.php line 12)
```

Comment:
In new sql file, you should not need to do step 4 to 6

Reply:
currently the sql file is still the old version, so we should add step 4 to 6
@TingYi88 will be responsible to delete step 4 to 6 while he uploads the new version of sql file